package.json

```diff
@@ -56,6 +56,7 @@
     "karma-safari-launcher": "^1.0.0",
     "karma-webpack": "^2.0.3",
     "mocha": "^3.4.2",
+    "nyc": "^11.0.2",
```

Comment:
@aqrln, but after looking at it one more time, the major version of this package in `package-lock.json` is 10, not 11, so it should still probably be updated accordingly.